pallets/nfts/src/lib.rs

```diff
@@ -474,12 +505,16 @@ pub mod pallet {
 		/// `collection` was cancelled by its `owner`.
 		ApprovalCancelled {
 			collection: T::CollectionId,
-			item: T::ItemId,
+			item: Option<T::ItemId>,
 			owner: T::AccountId,
 			delegate: T::AccountId,
 		},
-		/// All approvals of an item got cancelled.
-		AllApprovalsCancelled { collection: T::CollectionId, item: T::ItemId, owner: T::AccountId },
+		/// All approvals of a collection or item were cancelled.
```

Comment:
```suggestion
		/// All specified approvals of a collection or item were cancelled.
```
This is in relation to the fact that we dont know that we have cancelled ALL approvals, only all approvals up to some limit. Open to better suggestions.

Reply:
How about we change the event name to `ApprovalsCancelled` with a new field `approvals : u32` to demonstrate the number of removed approvals?